Web/hermes.client-web/src/store/modules/auth.js

```diff
@@ -0,0 +1,64 @@
+import AuthService from '../../services/auth.service';
+
+ export const auth = {
+   namespaced: true,
+  state:  {
+    status: { loggedIn: false }
+  },
+  actions: {
+    attemptLogin({ commit }, {email,password}) {
+      return AuthService.login({email,password})
+      .then(
+        response => {
```

Comment:
Mperdeytika edw. To `AuthService.login` kanei return eite to `response.data` eite `error`. Den kserw pws douleuei me 2 arguments :shrug: